README.md

```diff
@@ -87,7 +87,6 @@ More can be found in the docs, but here's a quick list of features which this pr
 - lockScreen()
 - isLocked()
 - shake()
-- complexFind()

```

Comment:
@rompic 
It is need to think about. Anyway it is not in scope of this PR. 
And anyway it should be removed if it useless.